packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -129,8 +141,9 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
       <Box
         aria-expanded={expanded}
         className='iui-header'
+        aria-disabled={disabled}
         tabIndex={0}
-        onClick={handleToggle}
+        onClick={() => !disabled && handleToggle()}
```

Comment:
This `disabled` check could be done inside `handleToggle`

Reply:
moved from onClick to handleToggle

Reply:
instead of ternary, it would be better to use an if statement and early return.

```js
if (disabled) {
  return
}

setExpanded(!expanded);
onToggle?.(!expanded);
```

Reply:
done